### ISSUE TITLE
Add support for showing difference when line endings are different.

### ISSUE THREAD

**Issue by BernH4** [will133/vim-dirdiff#38]
Currently when only the line endings differ (Unix \n and Windows \r\n for example) DirDiff says that the files differ but the lines are all collapsed (like when there is really no difference)

With normal vimdiff i can use following command when already inside diff:
 :windo e! ++ff=unix
 
 or before starting vimdiff:
$ vimdiff --cmd 'set fileformats=unix' file1 file2

dos2unix and unix2dos commands can be used to reproduce this issue.

**Comment by andrey-starodubtsev** [comment]
#51 is another issue on this topic

You can use let g:DirDiffAddArgs = '--strip-trailing-cr' to achieve that.
